password_security/models/res_users.py

```diff
@@ -21,6 +21,7 @@ class ResUsers(models.Model):
 
     password_write_date = fields.Datetime(
         'Last password update',
+        default=lambda s: fields.Datetime.now(),
```

Comment:
Shouldn't be this without final `()`?

Reply:
Derp yes I totally didn't need the lambda here. Good call